lib/puppetserver/ca/sign_action.rb

```diff
@@ -100,7 +100,7 @@ def sign_certs(certnames,settings)
       def get_all_certs(settings)
         result = get_certificate_statuses(settings)
 
-        unless result.is_a? Net::HTTPOK
+        unless result.code == 200
```

Comment:
should this be in a separate commit?

Reply:
Seems fine to me as is. It's a minor refactor.